src/idl_gen_csharp.cpp

```diff
@@ -1990,6 +1990,19 @@ class CSharpGenerator : public BaseGenerator {
           "Newtonsoft.Json.Formatting.Indented);\n";
       code += "  }\n";
     }
+    if (parser_.root_struct_def_ == &struct_def) {
+      code += "  public static " + class_name +
+              " DeserializeFromFlatBuffers(byte[] fbBuffer) {\n";
+      code += "    return " + struct_def.name + ".GetRootAs" + struct_def.name +
+              "(new ByteBuffer(fbBuffer)).UnPack();\n";
+      code += "  }\n";
+      code += "  public byte[] SerializeToFlatBuffers() {\n";
+      code += "    var fbb = new FlatBufferBuilder(1);\n";
```

Comment:
`1` is a really bad default size, it is going to spend a lot of time re-allocating buffers this way. Probably anywhere from 4 to 64 K would be entirely reasonable given that this is very temporary.

Reply:
OK, set the initial buffer size to 64K.